components/clarity-vscode/server/src/serverBrowser.ts

```diff
@@ -23,7 +23,7 @@ declare const __EXTENSION_URL__: string;
     new BrowserMessageWriter(self),
   );
 
-  initSync(await wasmModule);
+  initSync({ module: await wasmModule });
```

Comment:
fixes a warning i saw in the console. Not mandatory but will probably be needed in the future